xml/en/enterprise.xml

```diff
@@ -0,0 +1,70 @@
+<!--
+  Copyright (C) Igor Sysoev
+  Copyright (C) Nginx, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../dtd/article.dtd">
+
+<article name="nginx: enterprise"
+         link="/en/enterprise.html"
+         lang="en"
+         rev="1">
+
+<section id="sponsors"
+        name="Corporate Sponsors and Ownership">
+
+<para>
+<link url="https://www.f5.com/">F5, Inc.</link>
+is the primary maintainer, sponsor, and steward of NGINX since the
+<link url="https://www.f5.com/company/news/press-releases/f5-acquires-nginx-to-bridge-netops-devops">acquisition of NGINX, Inc. in 2019.</link>
+Learn more about F5's
+<link url="docs/governance.html">stewardship and goverance</link>
```

Comment:
`<link doc="docs/governance.xml">stewardship and governance</link>`
(link doc vs link url, typo in governance)